modules/serverless-installing-cli-linux-ibm-z-tarball.adoc

```diff
@@ -0,0 +1,35 @@
+[id="installing-cli-linux-ibm-z-tarball-{context}"]
+= Installing the "Knative CLI" for Linux on IBM Z and LinuxONE 
+
+For Linux distributions, you can download the `kn` CLI  directly as a `tar.gz` archive.
+
+.Procedure
+
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest[`kn` CLI].
+
+. Unpack the archive:
++
+[source,terminal]
+----
+$ tar -xf <file>
+----
+
+. Move the `kn` binary to a directory on your PATH.
+
+. To check your path, run:
++
+[source,terminal]
+----
+$ echo $PATH
+----
+
++
+[NOTE]
+====
+If you do not use {op-system-base} or {op-system-base}, ensure that *libc* is installed in a directory on your library path.
+If *libc* is not available, you might see the following error when you run CLI commands:
+[source,terminal]
```

Comment:
same as above, single line space before [source,terminal]

Reply:
there doesn't need to be a space for these when you're using + to add a command in a step, but there should be otherwise when commands are used in normal paragraphs like this